proto/o5/application/v1/application.proto

```diff
@@ -22,6 +22,14 @@ message Application {
   // infrastructure adapters, e.g. a service which sends emails using SES, the
   // o5 deployer itself, dante etc.
   AWSConfig aws_config = 12;
+
+  SidecarMessageOptions sidecar_message_options = 13;
```

Comment:
It's a nitpick of the name, but wanted to ask whether this should be `sidecar_config`? It'd only make sense to me if we'd potentially have non message specific config that would go to the sidecar, but I don't even know that we'll have that.

As for opt in, as long as the "empty" values remain no action, it'll remain opt in. Should be easy in this case by a check for options not being nil.